### DISCUSSION TITLE
Call to action

### DISCUSSION THREAD

**Discussion by rmccar** [ONSdigital/design-system#712] [Feedback]
Use this place to discuss [call to action](https://ons-design-system.netlify.com/components/call-to-action/) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by LaurenDowling** [comment]
For the call to action button, the icon is very large when CSS is removed.
![image](https://user-images.githubusercontent.com/108019533/217628473-86cdf958-7d7a-42e7-8ed9-6a929f1fcd71.png)
Gov define the width and height attributes within the svg. This ensures it remains the same size when css is removed.
